components/brave_shields/content/browser/ad_block_custom_filters_provider.h

```diff
@@ -43,6 +45,7 @@ class AdBlockCustomFiltersProvider : public AdBlockFiltersProvider {
 
  private:
   const raw_ptr<PrefService> local_state_;
+  bool developer_mode_enabled_ = false;
```

Comment:
not sure if this is done elsewhere but i think @ShivanKaul wanted developer mode enabled for users who have already set up a custom filter. (would be good to have a test for that case)

Reply:
It's done in ad_block_pref_service c-tor. Ok, I'll add the test